roles/get_image_hash/defaults/main.yml

```diff
@@ -3,20 +3,32 @@ controller_tag: "{{ ai_version }}"
 installer_agent_tag: "{{ ai_version }}"
 installer_tag: "{{ ai_version }}"
 
-images_to_get_hash_for:
-  release:
-    image: quay.io/openshift-release-dev/ocp-release
-    tag: "{{ release_tag | default(openshift_full_version + '-x86_64') }}"
+ocp_release_versions_map_all:
+  '4.6': 4.6.16
+  '4.7': 4.7.33
+  '4.8': 4.8.14
+  '4.9': 4.9.0
+
+get_release_images: true
+get_all_release_version: false
```

Comment:
I like these variable names (especially the `get_` prefix, we don't use that enough)! :+1: 
Was the var supposed to be named `get_all_release_versions`, though?

Reply:
Yes thanks :)